ocs_ci/ocs/resources/pod.py

```diff
@@ -985,3 +985,17 @@ def get_operator_pods(operator_label=constants.OPERATOR_LABEL, namespace=None):
     operators = get_pods_having_label(operator_label, namespace)
     operator_pods = [Pod(**operator) for operator in operators]
     return operator_pods
+
+
+def upload_to_pod(pod_name, localpath, remotepath):
```

Comment:
s/to_pod/ is redundant,  just pod.upload() should be sufficient.

Reply:
Done